usaspending_api/awards/management/commands/load_subawards_sql/030_frame_out_subawards.sql

```diff
@@ -88,6 +90,7 @@ select
       else null
     end,                                                    -- fain
     sub_ultimate_parent_unique_ide,                         -- parent_recipient_unique_id
+    sub_ultimate_parent_uei,                                -- parent_recipient_uei
```

Comment:
Adding Emily's comment from Slack here about wrapping this with UPPER(). Don't need to for DUNS because it is numerical.